inlang/packages/paraglide/paraglide-js/src/compiler/server/middleware.test.ts

```diff
@@ -373,6 +373,170 @@ test.skip("doesn't redirect if disableUrlRedirect is true", async () => {
 	expect(middlewareResolveWasCalled).toBe(true);
 });
 
+test("handles URLs with and without trailing slashes", async () => {
+	const runtime = await createParaglide({
+		project: await newProject({
+			settings: {
+				baseLocale: "en",
+				locales: ["en", "fr"],
+			},
+		}),
+		compilerOptions: {
+			strategy: ["url", "baseLocale"],
+			urlPatterns: [
+				{
+					pattern: "https://example.com/:path(.*)?",
```

Comment:
@axekan why would the middleware not execute if a path has a trailing slash or not?

Reply:
Or why is the middleware execution dependent on trailing slashes and non trailing slashes. The test seems reundant